src/openloco/windows/News/News.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+#include "../../company.h"
+#include "../../graphics/gfx.h"
+#include "../../interop/interop.hpp"
+#include "../../ui/WindowManager.h"
+
+using namespace openloco::interop;
+
+namespace openloco::ui::NewsWindow
+{
+    static loco_global<uint8_t[31][4], 0x004F8B08> _byte_4F8B08;
+    static loco_global<uint8_t[31][4], 0x004F8B09> _byte_4F8B09;
+    static loco_global<uint16_t[31], 0x004F8BE4> _word_4F8BE4;
+    static loco_global<uint8_t[31], 0x004F8C22> _messageTypes;
+    static loco_global<uint8_t[31], 0x004F8C41> _messageSounds;
+    static loco_global<uint8_t, 0x00508F1A> _gameSpeed;
+    static loco_global<uint8_t[3], 0x005215B5> _unk_5215B5;
+    static loco_global<uint32_t, 0x00523338> _cursorX2;
+    static loco_global<uint32_t, 0x0052333C> _cursorY2;
+    static loco_global<uint32_t, 0x00525CD0> _dword_525CD0;
+    static loco_global<uint32_t, 0x00525CD4> _dword_525CD4;
+    static loco_global<uint32_t, 0x00525CD8> _dword_525CD8;
+    static loco_global<uint32_t, 0x00525CDC> _dword_525CDC;
+    static loco_global<uint16_t, 0x00525CE0> _word_525CE0;
+    static loco_global<company_id_t, 0x00525E3C> _playerCompany;
+    static loco_global<uint16_t, 0x005271CE> _messageCount;
+    static loco_global<uint16_t, 0x005271D0> _activeMessageIndex;
+    static loco_global<int32_t, 0x00E3F0B8> gCurrentRotation;
+    static loco_global<int16_t, 0x112C876> _currentFontSpriteBase;
+    static loco_global<char[512], 0x0112CC04> byte_112CC04;
+    static loco_global<uint32_t, 0x011364EC> _numTrackTypeTabs;
+    static loco_global<int8_t[8], 0x011364F0> _trackTypesForTab;
+
+    namespace common
+    {
+        enum newsType
+        {
+            none = 0,
+            ticker,
+            newsWindow,
+        };
+
+        enum newsItem
+        {
+            majorCompany,
+            majorCompetitor,
+            minorCompany,
+            minorCompetitor,
+            general,
+            advice,
+        };
+
+        enum messageType
```

Comment:
Isn't this just `message_type` in `messagemgr.h`

Reply:
Yeah I’ve merged them in my uncommitted changes